charts/consul/values.yaml

```diff
@@ -86,7 +86,7 @@ global:
   # image that is used for functionality such as catalog sync.
   # This can be overridden per component.
   # @default: hashicorp/consul-k8s-control-plane:<latest version>
-  imageK8S: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3.5-dev
+  imageK8S: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.3-dev
```

Comment:
Should this be `consul-k8s-control-plane:1.3.6-dev`?

Reply:
The 1.4.x branch uses just 1.4-dev. We publish both so I think that is what we would want to do? Just use the 1.3-dev

Reply:
We publish both the 1.3-dev and 1.3.6-dev